Misc/NEWS.d/next/Core and Builtins/2020-01-25-23-51-17.bpo-39453.xCOkYk.rst

```diff
@@ -0,0 +1,2 @@
+Fix contains method of list to hold strong references of elements while
```

Comment:
It describes the implementation, but says nothing to an average Python user. I would formulate the entry in terms of Python: "Fixed possible crash in ``list.__contains__` when a list is changed during comparing items."

Reply:
Updated!